### DISCUSSION TITLE
Tooltip

### DISCUSSION THREAD

**Discussion by samprasdsouza** [xyflow/xyflow#2693] [Q&A] [answered]
I am new to react-flow , can anyone please guide on Implementing tooltip for node on hover/onClick .  I see there is an props for `onNodeMouseEnter` which return the id of node on Hover but how can generate a tooltip over selected node thanks :)

**Comment by moklick** [comment, marked as answer]
You can use the NodeToolbar for that https://reactflow.dev/docs/api/nodes/node-toolbar/

**Reply by AmmarZeer** [reply]
Do edges have something similar to NodeToolbar?

and if not, is there a way to use this component on edges?
